install.sh

```diff
@@ -53,6 +54,19 @@ tty_red="$(tty_mkbold 31)"
 tty_bold="$(tty_mkbold 39)"
 tty_reset="$(tty_escape 0)"
 
+have_sudo() {
```

Comment:
This naming (or `execute_sudo`) feels a bit strange given sudo is probably always available but just not accessible and that this returns true unconditionally on macOS.

Reply:
`have_sudo_access` might be a better fit here. Thoughts?

Reply:
Perfect!

Reply:
Thoughts: checking for access on macOS too makes sense, I think, and just erroring out if you don't have `sudo` there?